packages/exchange/src/exchange/exchange.py

```diff
@@ -143,24 +144,23 @@ def call_function(self, tool_use: ToolUse) -> ToolResult:
             return ToolResult(tool_use_id=tool_use.id, output=output, is_error=True)
 
         try:
-            if isinstance(tool_use.parameters, dict):
-                output = json.dumps(tool.function(**tool_use.parameters))
-            elif isinstance(tool_use.parameters, list):
-                output = json.dumps(tool.function(*tool_use.parameters))
+            if isinstance(tool_use.parameters, dict) or isinstance(tool_use.parameters, list):
+                function_return_value = tool.function(**tool_use.parameters)
```

Comment:
is this intentional? i think trying to unpack a `list` (`*`) will throw an error since it's positional arguments not keyword arguments